benchmarks/benchmark_dataset.py

```diff
@@ -0,0 +1,56 @@
+import torch
```

Comment:
@msbaines : should I put this file and the experimental_ampnet.py inside an benchmarks/experimental/ folder.

Looks like this file (benchmark_dataset.py) was removed earlier.

Reply:
drive by comment: Mentioned in another conversation, we refactored fairscale benchmarks and have common dataset loaders. We should try and use that unless it doesn't work for your use case.

Reply:
Yeah, that's a great suggestion. I plan to refactor this, but I will first make a PR with xpipe (which depends on the dataloader from this script). Once we merge that, I will plan to refactor.